lightning/src/ln/peer_handler.rs

```diff
@@ -1179,6 +1179,23 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, L: Deref> PeerManager<D
 		};
 	}
 
+	/// Disconnect a peer given its node id.
+	///
+	/// Set no_connection_possible to true to prevent any further connection with this peer,
+	/// force-closing any channels we have with it.
+	///
+	/// Beware with reentrancy issues about disconnect_socket!
```

Comment:
Can we state that "if a peer is connected, this will call `disconnect_socket` on the descriptor for the peer" - there's no reason to not provide more detail.

Reply:
Updated with comment.